index.php

```diff
@@ -0,0 +1,68 @@
+<?php
+$nazevParty = "Czechitas";
+$poradatel = "Ondra, Petr, Jirka";
+$misto = "Praha";
+$kapacita = 30;
+
+$ucastnici = explode("\n", file_get_contents("ucastnici.txt"));
```

Comment:
To mi prijde slozity tyhle volani uvnitr funkce, vzal bych to pres jednorazovou promennou.


```suggestion
$obsahSouboru = file_get_contents("ucastnici.txt");
$ucastnici = explode("\n", $obsahSouboru);
```